layers/+lang/ruby/packages.el

```diff
@@ -36,6 +36,7 @@
         seeing-is-believing
         smartparens
         rake
+        (lsp-ruby :requires lsp-mode lsp-ui company-lsp)
```

Comment:
This package is not needed anymore according to https://github.com/emacs-lsp/lsp-ruby
<img width="698" alt="screen shot 2019-01-05 at 13 45 19" src="https://user-images.githubusercontent.com/120457/50724585-55408d00-10f0-11e9-944d-d4bd3dc46439.png">